buf.yaml

```diff
@@ -15,9 +15,7 @@ lint:
     - SERVICE_SUFFIX
     - PACKAGE_VERSION_SUFFIX
   ignore:
-    - tendermint
-    - gogoproto
-    - cosmos_proto
+    - third_party/proto
```

Comment:
wait, does this work? I though i got some proto lint errors with this.

https://github.com/cosmos/cosmos-sdk/pull/6905#discussion_r465599631

Reply:
ah didn't see this, thanks for linking the comment